### PR TITLE
Fix Create Call Signature

### PR DESCRIPTION
**Pull request by takameyer**
## What, How & Why?
This allows the create static function to be used without a type error.  This reflects the change we made in bindgen.

## ☑️ ToDos

* [ ] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 📝 Update `COMPATIBILITY.md`
* [ ] 🚦 Tests
* [ ] 🔀 Executed flexible sync tests locally if modifying flexible sync
* [ ] 📦 Updated internal package version in consuming `package.json`s (if updating internal packages)
* [ ] 📱 Check the React Native/other sample apps work if necessary
* [ ] 📝 Public documentation PR created or is not necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [ ] typescript definitions file is updated
* [ ] jsdoc files updated

### CODE REVIEW

**Comment by takameyer** [comment]
@kneth Can we get this in the next release?  Then I can update the templates to not use the constructor when creating realm objects

**Review by kraenhansen — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,7 @@
 * None
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None
+* Fix type error when using `realm.create` in combination with class base models.
```
Comment (kneth) [inline CHANGELOG.md:10]: Since v11.0.0?

**Review by kneth — APPROVED** [review]

**Comment by kneth** [comment]
@takameyer 

> Can we get this in the next release?

If merged today, I will include it in today's release 😉

File: CHANGELOG.md (hunk 1 above)
Comment (takameyer) [reply CHANGELOG.md:10]: Yes, I think so

**Pull request merged**